docs/source/faq.rst

```diff
@@ -494,3 +494,42 @@ There are two kinds of constrained optimizations, one with soft constraints and
 Soft constraints do not have to be satisfied, but an objective function is penalized if they are unsatisfied. On the other hand, hard constraints must be satisfied.
 
 Optuna is adopting the soft one and **DOES NOT** support the hard one. In other words, Optuna **DOES NOT** have built-in samplers for the hard constraints.
+
+How can I parallelize optimization?
+-----------------------------------
+
+The variations of parallelization are in the following three cases.
+
+1. Multi-threading parallelization with single node
+2. Multi-processing parallelization with single node
+3. Multi-processing parallelization with multiple nodes
+
+1. Multi-threading parallelization with a single node
+=====================================================
+
+Parallelization can be achieved by setting the argument ``n_jobs`` in :func:`~optuna.study.Study.optimize`.
+However, the python code will not be faster due to GIL because :func:`~optuna.study.Study.optimize` with ``n_jobs!=1`` uses multi-threading. 
+
+While optimizing, it will be faster in limited situations, such as waiting for other server requests, or waiting for C/C++ processing with numpy, etc., but it is not expected to be faster in other cases.
```

Comment:
```suggestion
While optimizing, it will be faster in limited situations, such as waiting for other server requests or C/C++ processing with numpy, etc., but it will not be faster in other cases.
```